### ISSUE TITLE
Downloading from a playlist show only one item

### ISSUE THREAD

**Issue by syedmahm** [jely2002/youtube-dl-gui#44]
Hello,

When I try to download from a playlist, I do not see the list of videos showing, rather it only shows one item and tells how many videos are in the playlist. It is still possible to download all the videos, but it is not possible to choose selected ones. 

I am trying to download this playlist:
https://www.youtube.com/playlist?list=PLB0j2hqgnW6UEIjZ6gFpgQXBdE0tayAlk

Please see the screenshot:
![Screen Shot 2021-04-22 at 4 46 19 PM](https://user-images.githubusercontent.com/18193637/115794180-c91f0600-a38a-11eb-9936-cb2d61561864.png)

Thanks!

**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Additional info (please complete the following information):**
 - OS: [e.g. Windows 10]
 - Application version [e.g. 1.8.4] (the version can be found in the settings menu)

**Additional context**

**Comment by jely2002** [comment]
Hi syedmahm,
Imagine adding a playlist with 1000 videos, showing every video in a list will slow down the application by a lot. This is why I decided to merge all the videos into one video representing the playlist. This is what's happening with your playlist right now. The default setting is that playlists bigger than 50 videos get merged into one, so by default my app should list all the videos of your playlist. 

My guess is that you changed the "Split playlists into separate videos when:" setting to "never split". This causes all videos in the playlists to get merged into one, regardless of length. If you change this setting back to "less than 50 videos" my app should list all videos in the playlist as long as it contains less than 50 videos.

This is how the setting should look like:
![illustration of the right setting](https://i.imgur.com/gK8ZzNR.png)

**TL;DR**
I believe you have the "Split playlists into separate videos when:" setting set to "never split". While it should be "less than 50 videos". As illustrated above.

Please let me know if you happen to have more questions.

**Comment by syedmahm** [comment]
Thank you Jelle for your quick and helpful reply. Is there a way I can select videos from a playlist. For example if I want to download video #6, 10-14, #19 etc.? 

Thanks again,Syed

    On Thursday, April 22, 2021, 5:04:05 PM MDT, Jelle Glebbeek ***@***.***> wrote:  
 
 


Hi syedmahm,
Imagine adding a playlist with 1000 videos, showing every video in a list will slow down the application by a lot. This is why I decided to merge all the videos into one video representing the playlist. This is what's happening with your playlist right now. The default setting is that playlists bigger than 50 videos get merged into one, so by default my app should list all the videos of your playlist.

My guess is that you changed the "Split playlists into separate videos when:" setting to "never split". This causes all videos in the playlists to get merged into one, regardless of length. If you change this setting back to "less than 50 videos" my app should list all videos in the playlist as long as it contains less than 50 videos.

This is how the setting should look like:


TL;DR
I believe you have the "Split playlists into separate videos when:" setting set to "never split". While it should be "less than 50 videos". As illustrated above.

Please let me know if you happen to have more questions.

**Comment by jely2002** [comment]
Thanks for the follow-up question,
I have removed the feature that allows selection of videos to download by their number, in favour of listing all videos in a playlist. This is because youtube-dl-gui now supports more services than only YouTube. Playlists are not always ordered the same way when requested by youtube-dl as when the user views them on the service itself. This can cause confusion because with a different order, number 16 can be a different video than expected.
